meta-resin-common/recipes-core/util-linux/files/0001-libblkid-don-t-check-for-size-on-UBI-char-dev.patch

```diff
@@ -0,0 +1,27 @@
+From 4360f56cf6e40b10ab6ace4cf74073de2381c192 Mon Sep 17 00:00:00 2001
+From: Karel Zak <kzak@redhat.com>
+Date: Tue, 11 Jul 2017 12:17:40 +0200
+Subject: [PATCH] libblkid: don't check for size on UBI (char dev)
+
+Signed-off-by: Karel Zak <kzak@redhat.com>
```

Comment:
Missing Upstream-Status just above your signed-off. Also add a blank line above the Upstream-Status when you add it